main.go

```diff
@@ -92,6 +92,12 @@ func mainE() error {
 		if err := checkout(gitCmd, checkoutArg, config.CloneDepth); err != nil {
 			return fmt.Errorf("checkout (%s): %v", checkoutArg, err)
 		}
+		// Update branch: 'git fetch' followed by a 'git merge' is the same as 'git pull'.
+		if config.Commit == "" && config.Branch != "" {
+			if err := run(gitCmd.Merge("origin/" + config.Branch)); err != nil {
+				return fmt.Errorf("merge %q: %v", checkoutArg, err)
```

Comment:
`checkoutArg` used in the error but in the actual command `config.Branch` is used; should be unified